src/Menu/MenuItem.js

```diff
@@ -211,9 +221,9 @@ const onKeyDown = (
 };
 
 const variantIcons = {
-  danger: <IconDanger size="medium" />,
-  success: <IconSuccess size="medium" />,
-  warning: <IconWarning size="medium" />
+  danger: <IconDanger size={`${pxToEm(24)}`} />,
```

Comment:
Not sure that you need the string interpolation here.  ...and elsewhere.

```js
<IconDanger size={pxToEm(24)} />
```

Reply:
You’re right. Addressed.